src/runtime/entitlements-manager.js

```diff
@@ -356,30 +371,59 @@ export class EntitlementsManager {
       .callbacks()
       .triggerEntitlementsResponse(Promise.resolve(entitlements));
 
+    const entitlement = entitlements.getEntitlementForThis();
+    if (entitlement) {
+      this.onAccessGranted_(entitlement);
+    }
+  }
+
+  /**
+   * @param {!Entitlement} entitlement
+   * @private
+   */
+  onAccessGranted_(entitlement) {
     // Show a toast if needed.
-    this.maybeShowToast_(entitlements);
+    this.maybeShowToast_(entitlement);
+    //TODO: if meter
+    setTimeout(() => {
+      this.markMeterAsUsed("signed meter"); //TODO
+    }, 2000);
   }
 
   /**
-   * @param {!Entitlements} entitlements
+   * @param {!string} signedMeter
+   */
+  markMeterAsUsed(signedMeter) {
+    if (isExperimentOn(this.win_, ExperimentFlags.METERING)) {
+      const url =
+        '/publication/' +
+        encodeURIComponent(this.publicationId_) +
+        '/entitlements';
+      const message = new EntitlementsPingbackRequest();
+      message.setSignedMeter(signedMeter);
+      message.setClientEventTime(toTimestamp(new Date()));
+      message.setHashedCanonicalUrl(this.url_);
```

Comment:
For now, I would suggest just sending a dummy string for the hashed URL and publisher user ID. We don't know where these values will come from yet, but I'm working on designing a solution :)